copulas/multivariate/gaussian.py

```diff
@@ -4,22 +4,27 @@
 import pandas as pd
 from scipy import integrate, stats
 
-from copulas import get_qualified_name
+from copulas import get_qualified_name, import_object
 from copulas.multivariate.base import Multivariate
-from copulas.univariate import GaussianUnivariate, Univariate
+from copulas.univariate import Univariate
 
 LOGGER = logging.getLogger(__name__)
 
 
 class GaussianMultivariate(Multivariate):
-    """Class for a gaussian copula model."""
+    """Class for a gaussian copula model.
 
-    def __init__(self):
+    Args:
+        distribution (str): Full qualified name of the class to be used as distribution.
+    """
+
+    def __init__(self, distribution=None):
         super().__init__()
 
         self.distribs = {}
         self.covariance = None
         self.means = None
+        self.distribution = distribution or 'copulas.univariate.gaussian.GaussianUnivariate'
```

Comment:
This would be better set as a module constant and as the default value for the argument:
```
DEFAULT_DISTRIBUTION = '...'
...
class ...:
    def __init__(self, distribution=DEFAULT_DISTRIBUTION):
        ...
```